src/audits/ImageWithoutAltText.js

```diff
@@ -27,7 +28,15 @@ axs.AuditRules.addRule({
             !axs.utils.isElementOrAncestorHidden(element);
     },
     test: function(image) {
-        return (!image.hasAttribute('alt') && image.getAttribute('role') != 'presentation');
+        var imageIsPresentational = (image.hasAttribute('alt') && image.alt == '') || image.getAttribute('role') == 'presentation';
+        if (imageIsPresentational)
+            return false;
```

Comment:
Question - should this be in the matcher? In other words should presentational images show a "Pass" (yes it has text alternative) or "NA" (not relevant to this test)?